src/core/src/rt_info.cpp

```diff
@@ -9,7 +9,23 @@
 
 namespace {
 
-ngraph::Node::RTMap mergeRuntimeInfo(const ngraph::NodeVector& nodes) {
+std::unordered_map<std::string, std::vector<ov::Any>> get_copyable_attrs(const ngraph::OutputVector& outputs) {
+    std::unordered_map<std::string, std::vector<ov::Any>> attrs;
+    for (const auto& output : outputs) {
+        for (const auto& item : output.get_rt_info()) {
+            bool copy = true;
+            if (item.second.is<ov::RuntimeAttribute>()) {
+                copy = item.second.as<ov::RuntimeAttribute>().is_copyable();
+            }
+            if (copy) {
+                attrs[item.first].push_back(item.second);
```

Comment:
just a question: I'm just worried about API generalization: should we check if the attribute exists already and merge if exists? May be additional input argument with default value would be appropriate here?